aws_topology/stackstate_checks/aws_topology/resources/kinesis.py

```diff
@@ -1,88 +1,118 @@
-from .utils import make_valid_data, create_arn as arn
+from collections import namedtuple
+from .utils import (
+    make_valid_data,
+    create_arn as arn,
+    client_array_operation,
+    set_required_access_v2,
+    transformation,
+)
+from schematics import Model
+from schematics.types import StringType, ModelType
 from .registry import RegisteredResourceCollector
 
 
 def create_arn(region=None, account_id=None, resource_id=None, **kwargs):
-    return arn(resource='kinesis', region=region, account_id=account_id, resource_id='stream/' + resource_id)
+    return arn(resource="kinesis", region=region, account_id=account_id, resource_id="stream/" + resource_id)
+
+
+StreamData = namedtuple("StreamData", ["stream", "tags"])
+
+
+class Stream(Model):
+    StreamName = StringType()
+    StreamARN = StringType()
+
+
+class StreamDescriptionSummary(Model):
+    StreamDescriptionSummary = ModelType(Stream)
```

Comment:
This one should be dereferenced in the collect_stream_description
Saves one schematics class

Reply:
Will put a todo since the format does not match the other resources, as shown in stackpack